tests/test_ops/test_load_ontology.py

```diff
@@ -0,0 +1,135 @@
+import os
+import pytest
+import tempfile
+import shutil
+from unittest.mock import patch, MagicMock
+from dagster import build_op_context
+from nmdc_runtime.site.resources import mongo_resource
+from nmdc_runtime.site.ops import load_ontology
+
+
+@pytest.fixture
+def client_config():
+    # Print details about the MongoDB configuration for debugging
+    mongo_host = os.getenv("MONGO_HOST")
+    mongo_dbname = os.getenv("MONGO_DBNAME")
+    mongo_username = os.getenv("MONGO_USERNAME")
+    
+    print(f"Test MongoDB connection details:")
+    print(f"- MONGO_HOST: {mongo_host}")
+    print(f"- MONGO_DBNAME: {mongo_dbname}")
+    print(f"- MONGO_USERNAME: {mongo_username}")
+    
+    # For local development outside Docker, try connecting to the Docker-exposed port
+    if mongo_host == "mongodb://mongo:27017":
+        alternative_host = "mongodb://localhost:27018"
+        print(f"- Inside test: MongoDB host is set to container name. "
+              f"If running test locally (not in Docker), try: {alternative_host}")
+    
+    return {
+        "dbname": mongo_dbname,
+        "host": mongo_host,
+        "password": os.getenv("MONGO_PASSWORD"),
+        "username": mongo_username,
+    }
+
+
+@pytest.fixture
+def temp_directory():
+    # Create a temporary directory for test output
+    temp_dir = tempfile.mkdtemp()
+    yield temp_dir
+    # Clean up after test
+    shutil.rmtree(temp_dir)
```

Comment:
Not needed. Use `pytest`'s built-in `tmp_path` fixture: https://docs.pytest.org/en/stable/how-to/tmp_path.html#the-tmp-path-fixture

Reply:
done.